### PR TITLE
fix: "onload" event order

### PR DESCRIPTION
**Pull request by benjamingr**
Deno has a bug where it currently always attaches `.onload` before `addEventListener` see [this](http://jsbin.testim.io/wer/1/edit?html,js,output) for a web example.

The following code:
```js
let arr: any = [];
window.addEventListener('load', () => {
  arr.push(1);
});
window.onload = () => arr.push(2);
window.addEventListener('load', () => {
  arr.push(3);
});

window.addEventListener('load', () => {
  setTimeout(() => {
    console.log(arr);
  })
})
```

Logs `[ 2, 1, 3 ]` in deno whereas browsers (and deno with this patch) log `[ 1, 2, 3 ]`

Note that this can be a relatively risky change compared to the AbortSignal/WebSocket/Worker timing fixes

### CODE REVIEW

File: cli/rt/99_main.js [hunk 1]
```diff
@@ -283,20 +283,45 @@ delete Object.prototype.__proto__;
...
+          } else {
+            handlerWrapper = makeWrappedHandler(value);
+            this.addEventListener(name, handlerWrapper);
+          }
+          this[handlerSymbol].set(name, handlerWrapper);
+        },
+        configurable: true,
+        enumerable: true,
+      });
+    }
```
Comment (bartlomieju) [inline cli/rt/99_main.js:322]: It seems this code can be obtained using `const { defineEventHandler } = window.__bootstrap.webUtil;`

**Review by bartlomieju — CHANGES_REQUESTED** [review]

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks @benjamingr

**Comment by benjamingr** [comment]
Thanks, sorry I'm a bit sick (and on my phone and not computer) - note that de-duplication here means that we return `undefined` and not `null` for things like `onload` which _probably_ needs a separate fix.

**Comment by bartlomieju** [comment]
> Thanks, sorry I'm a bit sick (and on my phone and not computer) - note that de-duplication here means that we return `undefined` and not `null` for things like `onload` which _probably_ needs a separate fix.

@benjamingr thanks for pointing this out, should we hold on before merging then?

**Comment by benjamingr** [comment]
> @benjamingr thanks for pointing this out, should we hold on before merging then?

I think merging is probably fine it's an improvement I will follow up with another fix

**Pull request merged**